coalesce/organizers/views.py

```diff
@@ -0,0 +1,14 @@
+from rest_framework import viewsets, mixins
+from .models import Organizer
+from rest_framework.permissions import IsAuthenticated
+from .serializers import CreateOrganizerSerializer
+
+
+class OrganizerCreateViewSet(mixins.CreateModelMixin,
```

Comment:
I wonder if this view should also create a corresponding user as one cannot exist without the other. I think it's more likely that we will have entirely separate user creation flows based upon whether that user is a volunteer or an organizer

Reply:
As per our discussion in channel: https://chat.federationof.tech/coalesce/pl/733h349n1fyodkdz3iwcfo9iaa

I'm going to rescind my point here and go ahead and merge :)